app/mailers/mailer.rb

```diff
@@ -16,4 +16,10 @@ def order_shipped(order)
       to: @order.email
     )
   end
+
+  private
+
+  def denise
+    'denise@radfordsofsomerford.co.uk'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.